viewing_party/party.py

```diff
@@ -1,23 +1,181 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_info = {}
+    if not title or not genre or not rating:
+        return None
+
+    # if title and genre and rating:
+
+    movie_info["title"] = title
+    movie_info["genre"] = genre
+    movie_info["rating"] = rating

```

Comment:
While manually creating our dictionary is great, don't be afraid to return a hard coded dictionary instead!